README.md

```diff
@@ -33,17 +33,35 @@ To do this you first need to output the benchmark results in the `.dat` format:
 cargo run --release -- --dat results.dat
 ```
 
-After having generated the `.dat` file, you can then pass it to the `gnuplot.sh`
-script to generate the SVG plot:
+After having generated the `.dat` file, you can then pass it to a script in the
+`./gnuplot` directory script to generate the SVG plot:
 
 ```
-./gnuplot.sh results.dat output.svg
+./gnuplot/summary.sh results.dat output.svg
 ```
 
 You can combine any number of results by passing them to the gnuplot script:
 
 ```
-./gnuplot.sh *.dat output.svg
+./gnuplot/summary.sh *.dat output.svg
+```
+
+And can plot detailed results using `detailed.sh`:
+
+```
+mkdir output/
+./gnuplot/detailed.sh *.dat output/
+```
+
+### Requirements
+
+The scripts for plotting the benchmark data with `gnuplot` uses negative
+indexes for substrings. This was added in bash
+[`4.2`](https://superuser.com/questions/1033273/bash-4-3-substring-negative-length-on-os-x#comment1442181_1033273).
```

Comment:
It'd be nice if here I could link to the actual docs that introduce it but I couldn't find it quickly

Reply:
I did find [this](https://github.com/bminor/bash/blob/bash-4.2/CHANGES#L341) but I don't know if that's any better really

Reply:
You said you fixed this in your local version, why should we add a comment about the minimum version rather than just changing the code to support older versions?

Reply:
No reason :-) I'm fine with either way! I'll go ahead and do the updates instead 👍

Reply:
4c3027f